src/format/output.js

```diff
@@ -206,6 +206,24 @@ function outReceipt (receipt) {
   return receipt;
 }
 
+function outPrivateReceipt (receipt) {
```

Comment:
Small detail: can the functions in this file be ordered by alphabetical order? outPrivateReceipt should go just before outReceipt

Reply:
Sure